src/main/java/core/basesyntax/service/FruitTransactionParser.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface FruitTransactionParser {
+    List<FruitTransaction> toTransaction(List<String> data);

```

Comment:
```suggestion
public interface DataParserService {
    List<FruitTransaction> parseData(List<String> data);
```
we parse data here. fruitTransactions is a result